Mathlib/SetTheory/Game/Short.lean

```diff
@@ -274,36 +274,36 @@ instance shortBit1 (x : PGame.{u}) [Short x] : Short (bit1 x) := by dsimp [bit1]
 We build `Decidable (x ≤ y)` and `Decidable (x ⧏ y)` in a simultaneous induction.
 Instances for the two projections separately are provided below.
 -/
-def leLfDecidable : ∀ (x y : PGame.{u}) [Short x] [Short y], Decidable (x ≤ y) × Decidable (x ⧏ y)
+def leLFDecidable : ∀ (x y : PGame.{u}) [Short x] [Short y], Decidable (x ≤ y) × Decidable (x ⧏ y)
```

Comment:
Not a big deal, but this name seems really quite weird. Do our naming conventions allow `lelfDecidable`? It might be worth asking on Zulip, but feel free to merge before getting an answer

bors d+